dockerfiles/init/modules/keycloak/templates/che-realm.json.erb

```diff
@@ -58,8 +58,18 @@
     "enabled" : true,
     "clientAuthenticatorType" : "client-secret",
     "secret" : "56abc663-1c41-4ab2-81d1-d4b704249221",
+<% if scope.lookupvar('che::che_single_port') == "true" -%>
```

Comment:
I'm sorry to say that but, unfortunately, these changes require some changes here: https://github.com/eclipse/che/blob/master/dockerfiles/init/s2i/keycloak/assemble#L42 
as you can see there for some reason we are removing all template injections because file `che-realm.json.erb` will be just renamed to `che-realm.json` and used in KC as is...

Reply:
Ok, yes, i forgot about this problem. I think we can remove this, since it is will be applied only if the first keycloak run is made with singleport option. This will require the redirect and webOrigin URL's to be set manually in keycloak.